### DISCUSSION TITLE
Unable to remote push using netbeans 11.1

### DISCUSSION THREAD

**Discussion by psusrini** [community/community#38880] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hi
I created a new account on github.
I have an older version of netbeans (11.1) from which I want to remote push my code into my repo

but it does not take my username and password (popup keeps repeating). Do I  need to configure my account to accept the push ?

**Comment by airtower-luna** [comment]
You cannot use your password for `git push` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).
